l2-frontend/src/main.js

```diff
@@ -76,6 +76,8 @@ new Vue({
     'DocCall': () => import('@/pages/DocCall'),
     'ListWait': () => import('@/pages/ListWait'),
     'LoadFile': () => import('@/ui-cards/LoadFile'),
+    'PlanPharmacoTherapy': () => import('@/pages/PlanPharmacoTherapy'),
```

Comment:
Надо везде переименовать (файл тоже)
```suggestion
    'PlanPharmacotherapy': () => import('@/pages/PlanPharmacotherapy'),
```
Правильно именно Pharmacotherapy